roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -1,8 +1,9 @@
 # Router Module
 
-The Router module is responsible for storing and dispatching Upward and Downward messages from and to parachains respectively. It is intended to later handle the XCMP logic as well.
+The Router module is responsible for all messaging mechanisms supported between paras and the relay chain, specifically: UMP, DMP, HRMP and later XCMP.
```

Comment:
I think it would be better to add a separate page for describing those in detail and linking this page here. As for now, I added those in the glossary.